src/containers/post/post.component.jsx

```diff
@@ -26,9 +27,26 @@ const Post = (props) => {
   const voteAPercent = (voteACount / (voteACount + voteBCount)) * 100;
   const voteBPercent = (voteBCount / (voteACount + voteBCount)) * 100;
   return (
-    <li className="rounded-lg pt-1 m-1 mt-3 bg-bluey ">
-      <h4 className="text-grayy ml-2">Username here</h4>
-      <h6 className="text-grayy p-2 mx-2 mb-2 rounded-lg border-2 border-grayy">
+    <li className="rounded-lg pt-1 m-1 mt-3 bg-bluey font-noto tracking-wide">
+      <div className="flex justify-between">
+        <div className="flex mb-1">
+          <img
+            src={props.postOwnerAvatar}
+            className="rounded-full h-6 w-6 align-center ml-2"
+          />
+          <h4 className="text-grayy ml-2">{props.postOwnerUsername}</h4>
+        </div>
+        <button onClick={() => setViewTotalVotes(!viewTotalVotes)}>
+          <i
+            className={`${
+              canUserViewVote ? " show " : " hidden "
```

Comment:
```suggestion
              canUserViewVote ? "show" : "hidden"
```

Reply:
Just do that sometimes because it fucks with the tailwind

Reply:
Done